pkg/ibmvpc/examples/out/explain_out/LBToResIPNode_all_vpcs_explain_debug.txt

```diff
@@ -0,0 +1,296 @@
+Explaining connectivity from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca to 192.168.32.5 (iks-clusterid:1[192.168.32.5]) within ky-test-vpc
+=================================================================================================================================================
+
+No connections are allowed from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca[LB private IP][192.168.36.6] to iks-clusterid:1[192.168.32.5];
+connection is blocked by Load Balancer
+
+Load Balancer: kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca blocks connection to iks-clusterid:1[192.168.32.5] which is not one of its pool members
+Egress: security group kube-r006-d7cfb31a-1d4b-40c8-83df-ce2e6f8f2e57 allows connection; network ACL ky-test-edge-acl allows connection
+Ingress: network ACL ky-test-edge-acl allows connection; security group kube-r006-d7cfb31a-1d4b-40c8-83df-ce2e6f8f2e57 allows connection
+
+Path:
+	kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca[LB private IP][192.168.36.6] -> |
```

Comment:
in this case maybe we do not need to split the output per private IP ?
the user asks about LB, and if the dest is not a pool member, it is a common explanation why the connection is blocked for all private IPs ?
or add initial such explanation for the LB itself, and then per private IP the explanation per sg/acl ?

Reply:
the load balancer is is being treat as all other nodeSets, I didnt find an explanability example, so I added multiNIsToSingleNI.
I also played with vpe "iks-clusterid:1" at iks_config_object example
in such cases, the explanation is being split to the different NIs, even when the explanation is the same for all NIs/ResIp
there is a need to group explanations, but since we want to show all the relevant rules, the solution can not be load balancer specific

Reply:
please open a separate issue so this can be considered there.

Reply:
https://github.com/np-guard/vpc-network-config-analyzer/issues/674